### ISSUE TITLE
[BUG] Could not parse respons

### ISSUE THREAD

**Issue by raniaamina** [FujiwaraChoki/MoneyPrinter#35]
**Describe the bug**
Script error after generating audio and on process get video from pexel

**To Reproduce**
1. Just doing step like in docs
2. Wait until audio generated
3. Error appear

 
**Expected behavior**
Video generated

**Screenshots**
![gambar](https://github.com/FujiwaraChoki/MoneyPrinter/assets/22146607/6c98cb5c-baf5-486c-b0c1-aa1b65fa4cc1)

**Desktop (please complete the following information):**
 - OS: Linux
 - Browser: Firefox
 - Python Version 3.11.7

**Additional context**
backend error log

```bash
[*] GPT returned an unformatted response. Attempting to clean...
[-] Could not parse response.
[-] Error: the JSON object must be str, bytes or bytearray, not list
127.0.0.1 - - [07/Feb/2024 23:27:36] "POST /api/generate HTTP/1.1" 200 -

```

**Comment by jensblond** [comment]
Try `pip3 install PyObjC`

**Comment by FujiwaraChoki** [comment]
This just happens sometimes, I suggest you restart the webserver.

**Comment by raniaamina** [comment]
Let me try again

**Comment by raniaamina** [comment]
The last error message is gone!

But it still failed; and after i recheck vimeo is blocked ISP :")
using vpn is solve this issue
